boards/v2/qemu/qemu_x86/board.yml

```diff
@@ -0,0 +1,30 @@
+boards:
+
+  - name: qemu_x86
+    socs:
+      - name: ia32
+        variants:
+        - name: 'nokpti'
+        - name: 'nommu'
+        - name: 'nopae'
+        - name: 'virt'
+        - name: 'xip'
+
+  - name: qemu_x86_lakemont
+    socs:
+      - name: lakemont
+
+  - name: qemu_x86_64
+    socs:
+      - name: ia32
+        variants:
+        - name: 'nokpti'
+
+  - name: qemu_x86_tiny
+    socs:
+      - name: ia32
+    revision:
+      format: custom
+      default: ''
+      revisions:
+        - name: '768'
```

Comment:
if revisions are numbers, then why not  use `revision: format: number` ?

Reply:
This should be a variant then, not a revision

Reply:
changed to a variant